tools/README.md

```diff
@@ -50,4 +50,25 @@ $ python tools/tokenizer.py --text_input_path raw_data.txt --bin_output_path cn/
 
 例如，对于第一个`sequence`，`starting index`为 0，有 11 个`tokens`；对于第二个`sequence`，由于第一个`sequence`转换为`string`后的长度为`89`，因此它的`starting index`为 90，有 15 个`tokens`。
 
-`json`和`jsonl`类型的文件的`bin`和`meta`文件格式和`txt`一致，此处不再赘叙。
\ No newline at end of file
+`json`和`jsonl`类型的文件的`bin`和`meta`文件格式和`txt`一致，此处不再赘叙。
+
+# pal_inference.py
+在[GSM8K](https://huggingface.co/datasets/gsm8k)数据集上使用[PAL](https://github.com/reasoning-machines/pal)范式推理，使模型编写代码并通过Python解释器执行来解决数学问题。以下是使用示例：
+```bash
```

Comment:
Seems we have multi-args in the script, so we should give all the meanings of the parameter.